docs/reason.md

```diff
@@ -116,6 +116,15 @@ robot reason --input unreasoned.owl
   --output reasoned.owl
 ```
 
+Note that for some of the axiom generators you may need the Hermit reasoner
+to get meaningful results:
+
+```
```

Comment:
In order for this to be included in the integration test, it should use the `    ` (four spaces) code format, not the backticks, otherwise it gets skipped.

Reply:
Ooops I guess I was thrown because some of the code samples in reason.md actually use the triple-tick format, probably in order to not be run as part of the test suite.